src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -93,6 +93,8 @@ module.exports = overrideAll({
     ticks: axesAttrs.ticks,
     mirror: axesAttrs.mirror,
     ticklen: axesAttrs.ticklen,
+    ticklabelshiftx: axesAttrs.ticklabelshiftx,
+    ticklabelshifty: axesAttrs.ticklabelshifty,
```

Comment:
Please see my comment here: https://github.com/plotly/plotly.js/pull/7006/files#r1622582974

Reply:
See comment https://github.com/plotly/plotly.js/pull/7006#discussion_r1624655475
I removed the new properties from non-cartesian plots.